server/datastore/mysql/software.go

```diff
@@ -2262,10 +2269,33 @@ func (ds *Datastore) ListHostSoftware(ctx context.Context, host *fleet.Host, opt
 	}
 
 	var onlyVulnerableJoin string
+	var cveMetaJoin string
+	var vulnerabilityFiltersClause string
+	var cveArgs []any
```

Comment:
It looks like the `cveArgs` slice is only being used to detect if we want to add the `cveMetaJoin` to `onlyVulnerableJoin`. Any reason not to just use a `bool`? Am I missing something?

Reply:
We could simplify and use a boolean

Reply:
done!